src/app/src/main/kotlin/org/vaccineimpact/api/app/repositories/jooq/JooqTouchstoneRepository.kt

```diff
@@ -288,6 +303,15 @@ class JooqTouchstoneRepository(
             mapper.mapEnum(record.status)
     )
 
+    private fun mapRecordToTouchstone(record: Record) = Touchstone(
+            record[TOUCHSTONE.ID],
```

Comment:
there's already a method that does this - it's in this class, called `mapTouchstone`